framework/entity/src/main/java/org/apache/ofbiz/entity/GenericDelegator.java

```diff
@@ -1625,8 +1625,8 @@ public EntityListIterator findListIteratorByCondition(DynamicViewEntity dynamicV
      * @see org.apache.ofbiz.entity.Delegator#findCountByCondition(java.lang.String, org.apache.ofbiz.entity.condition.EntityCondition, org.apache.ofbiz.entity.condition.EntityCondition, org.apache.ofbiz.entity.util.EntityFindOptions)
      */
     @Override
-    public long findCountByCondition(String entityName, EntityCondition whereEntityCondition,
-            EntityCondition havingEntityCondition, EntityFindOptions findOptions) throws GenericEntityException {
+    public long findCountByCondition(String entityName, EntityCondition whereEntityCondition, Set<String> fieldsToSelect
```

Comment:
@verma-pawan I think we need to keep the existing method signature as well, and can pass null as fieldsToSelect to maintain backward compatibility.

Reply:
Thanks, @dixitdeepak totally agree. It slips off from my mind, on it.

Reply:
@dixitdeepak Done, please have look!